src/api/models/user.js

```diff
@@ -2,12 +2,27 @@ const user = (sequelize, DataTypes) => {
   const User = sequelize.define('Users', {
     username: {
       type: DataTypes.STRING,
+      allowNull: true
     },
     email: {
```

Comment:
Same for the email @Ramadhan0

Reply:
@Ramadhan0 the feedback @nkpremices gave above doesn't seem to have been addressed.